docs/available-commands.md

```diff
@@ -81,6 +81,8 @@ Sends text to a named terminal, which is then executed in the terminal. If the n
 
 Note: In order to send multiple commands spread out across multiple links in the same terminal window, ensure that the `name` passed is consistent.
 
+Note: On Windows, strings (such as for `echo` statements) must be enclosed in double quotes like `echo "my long string"` since the change with https://code.visualstudio.com/updates/v1_57#_native-line-wrapping-support-on-windows.
```

Comment:
it can also be single-quotes https://devblogs.microsoft.com/scripting/weekend-scripter-understanding-quotation-marks-in-powershell/#:~:text=In%20Windows%20PowerShell%2C%20single%20and,%E2%80%9Chere%20is%20my%20end.%E2%80%9D&text=Notice%20that%20in%20the%20last,instead%20of%20a%20string%20value.